tests/boilerplate_client/errors.py

```diff
@@ -0,0 +1,16 @@
+ERRORS = {
```

Comment:
I find it weird to have errors declared as a dict and not an enum here.
Maybe we could change it to an enum, and convert the enum to a dict where needed (only `test_status_word.py`?) with somehting like this : https://stackoverflow.com/a/60451617/5920310.
What do you think about it?

Reply:
Changed to enum